frontend/apps/app/app/api/chat/route.ts

```diff
@@ -16,6 +15,9 @@ const chatRequestSchema = v.object({
   isDeepModelingEnabled: v.optional(v.boolean(), true),
 })
 
+// https://vercel.com/docs/functions/configuring-functions/duration#maximum-duration-for-different-runtimes
+export const maxDuration = 300
```

Comment:
I would really like to go to 800 seconds, but that would require activating Fluid Compute. I would like to proceed separately.